common/views/components/FilterDrawerRefine/FilterDrawerRefine.js

```diff
@@ -175,321 +82,173 @@ const FilterDrawerRefine = ({ searchForm, searchParams }: Props) => {
-                      as="button"
-                      type="button"
-                      isActive={
-                        itemsLocationsLocationType &&
-                        onlineLocations.every(t =>
-                          itemsLocationsLocationType.includes(t)
-                        )
-                      }
-                      style={{ cursor: 'pointer' }}
-                    >
-                      online
-                    </ProtoTag>
-                  </NextLink>
-                  <NextLink
-                    passHref
-                    {...worksUrl({
-                      ...searchParams,
-                      page: 1,
-                      itemsLocationsLocationType: updateLocations(
-                        itemsLocationsLocationType,
-                        'library'
-                      ),
-                    })}
-                  >
-                    <ProtoTag
-                      as="button"
-                      type="button"
-                      isActive={
-                        itemsLocationsLocationType &&
-                        inLibraryLocations.every(t =>
-                          itemsLocationsLocationType.includes(t)
-                        )
-                      }
-                      style={{ cursor: 'pointer' }}
-                    >
-                      in library
-                    </ProtoTag>
-                  </NextLink>
-                </div>
-              </div>
-            </>
-          )}
-          <Space
-            v={{ size: 'l', properties: ['margin-top'] }}
-            className="tokens"
-          >
-            {refineFiltersPrototype &&
-              allWorkTypes.map(subcategory => {
-                return (
-                  workType &&
-                  workType.includes(subcategory.letter) && (
-                    <NextLink
-                      key={subcategory.title}
-                      passHref
-                      {...worksUrl({
-                        ...searchParams,
-                        workType: updateWorkTypes(workType, subcategory),
-                        page: 1,
-                      })}
-                    >
-                      <ProtoTag as="a" isActive small>
-                        &times; {subcategory.title}
-                      </ProtoTag>
-                    </NextLink>
-                  )
-                );
-              })}
-            {(productionDatesFrom || productionDatesTo) && (
-              <div className={classNames({ [font('hnl', 5)]: true })}>
-                <Divider extraClasses={'divider--thin divider--pumice'} />
+                <Checkbox
+                  id={type.data.id}
+                  text={`${type.data.label} (${type.count})`}
+                  value={type.data.id}
+                  name={`workType`}
+                  checked={workTypeInUrlArray.includes(type.data.id)}
+                  onChange={event => {
+                    changeHandler();
```

Comment:
Instead of doing this - why not make them links? That way we don't have to hijack the form functionality?

Reply:
If they are links, I'm not sure how best to convey to assisted technology what they're doing, as the behaviour changes depending on what other filters are selected and whether you're checking or unchecking them. It feels like semantically being form inputs is right.

Reply:
Semantically it's a link that takes you to a result all of those worktypes?
Same as we might add it to the work page too?

Reply:
it's not just a link to those worktypes though, it's a link to those worktypes with possibly other filters applied too, or it could be a link removing those worktypes